tests/tests_test.go

```diff
@@ -0,0 +1,271 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	"github.com/konveyor/mig-controller/pkg/pods"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	labels"k8s.io/apimachinery/pkg/labels"
+	"path"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+	"time"
+)
+
+var ctx = context.Background()
+
+var _ = Describe("Running migmigration when storage, cluster and plan are ready", func() {
+	var (
+		migPlan     *migapi.MigPlan
+		migPlanName string
+		namespaces []string
+	)
+	BeforeEach(func() {
+		namespaces = []string{"ocp-41583-longpvcname"}
+		migPlan = NewMigPlan(namespaces, migPlanName)
+	})
+
+	JustBeforeEach(func() {
+		Expect(hostClient.Create(ctx, migPlan)).Should(Succeed())
+		Eventually(func() string {
+			fmt.Println("waiting for migPlan to be ready")
+			hostClient.Get(ctx, client.ObjectKey{Name: migPlanName, Namespace: MigrationNamespace}, migPlan)
+			l := len(migPlan.Status.Conditions.List)
+			if l > 0 {
+				return migPlan.Status.Conditions.List[l-1].Type
+			}
+			return ""
+		}, time.Minute*5, time.Second).Should(Equal("Ready"))
+	})
+
+	AfterEach(func() {
+		err = hostClient.Delete(ctx, &migapi.MigPlan{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      migPlanName,
+				Namespace: MigrationNamespace,
+			},
+		})
+		if err != nil {
+			fmt.Println(err)
+		}
+	})
+
+	Describe("Testing BZ #1965421, PVC name is more than 63 char long", func() {
+		Context("Running a migration", func() {
+
+			BeforeEach(func() {
```

Comment:
What is the diff between this BeforeEach and the one above? 

https://github.com/konveyor/mig-controller/pull/1153/files#diff-e6060120fd60e25841122a6c7839035ef0b38ee3c856142ae3a864918b666b51R29

Reply:
The above one was a global one, but yeah we can get rid of the above one!